app/src/Util.re

```diff
@@ -0,0 +1,12 @@
+module CssHelper = {
+  open Css;
+  let (<<) = (a, b) => merge([a, b]);
+
+  let margin = (top, right, bottom, left) =>
```

Comment:
Would it make sense to have them named and with `0` defaults so that the caller can set them selectively?

Reply:
You mean like so?
```
let margin = (~top=0, ~right=0, ~bottom=0, ~left=0, ()) =>
  style([
    marginTop(px(top)),
    marginRight(px(right)),
    marginBottom(px(bottom)),
    marginLeft(px(left)),
  ]);
```
```
<Container style={margin(~top=48, ~bottom=24, ())}>
  <Container style={margin(~bottom=16, ())}>
    <Input placeholder="Enter your name" />
  </Container>
  <Input placeholder="Enter an avatar URL" />
</Container>
```

This could also work, I personally prefer the more concise form as it looks like raw css. I don't like that you have to add the empty unit `()` at the end for Reason to work.

I also think that the shorter form would lend itself better for composing styles inline, like so:
```
<Container style={ Styles.someBaseStyle << margin(0, 8, 0, 0) << Styles.moreOverrides}>
```

In the end I think it's a matter of taste. If you prefer the named style, we can do it.

Reply:
It's not purely a matter of taste as a set of 4 integers without any further explanation make it hard to deduce what each param is meant for. So in terms of ergonomics named params add a lot. The strictness of non-optional trailing parameters ruin this a bit for us for sure and as the order - top, right, bottom, left - is well known from CSS - I think we can get away with it for now.

Reply:
Cool, let's stick with the original version then.